h2o-r/h2o-package/R/communication.R

```diff
@@ -800,7 +800,12 @@ h2o.show_progress <- function() assign("PROGRESS_BAR", TRUE, .pkg.env)
       }
 
       job = jobs[[1]]
-
+      if(verboseModelScoringHistory){
```

Comment:
Yes, we can add it like this:
`if (keepRunning) {
        Sys.sleep(pollInterval)
        if(verboseModelScoringHistory){
          cat(paste0("\nScoring History for Model ",job$dest$name, " at ", Sys.time(),"\n"))
          print(paste0("Model Build is ", job$progress*100, "% done..."))
          print(tail(h2o.getModel(job$dest$name)@model$scoring_history))
        }
      } else {
        if (progressBar) {
          close(pb)
        }`